modules/wallet/persist.go

```diff
@@ -14,14 +17,6 @@ type savedKey struct {
 	Visible          bool
 }
 
-// legacySavedKey preserves compatibility with the other beta wallets. After
-// the full currency is launched, this struct and the related code can be
-// discarded.
-type legacySavedKey struct {
-	SecretKey        crypto.SecretKey
-	UnlockConditions types.UnlockConditions
-}
```

Comment:
I don't think I support this remove.

I know it's a bit gross but we need to maintain legacy support for at least a few versions.